Lesson5/Task1/src/Main.java

```diff
@@ -0,0 +1,159 @@
+public class Main {
+    public static void main(String[] args) {
+        Passenger passengerCar = new Passenger("KIA",210,210,3100,4,15,"Хечбек",6);
+        passengerCar.Description();
+        passengerCar.Counting(6);
+        System.out.println();
+        Cargo cargoCar = new Cargo("Маз",320,170,5800,8,17,4500);
+        cargoCar.Description();
+        cargoCar.Load();
+        System.out.println();
+        Civil civilAir = new Civil("WWW",540,810,4900,15,48,45,true);
+        civilAir.Description();
+        civilAir.loadPassengers();
+        System.out.println();
+        War warAir = new War("ИС",700,970,5050,13,34,true,6);
+        warAir.Description();
+        warAir.Shoot();
+        warAir.Ejection();
+    }
+}
+class Transport {
+    String brand;
```

Comment:
Используй модификаторы доступа.
```suggestion
    private String brand;
```